vnext/Microsoft.ReactNative.Cxx/CoreApp.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include <roapi.h>
```

Comment:
Why do we need this include here? The rest of the file does not seem to be using it.